docs/pages/reference/mesheryctl/subcommands/mesheryctl-pattern-apply.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: apply
+permalink: reference/mesheryctl/pattern/apply
+type: reference
```

Comment:
You've created and it's great but it must be reflected on the table also...

Reply:
Yes, it will be. Have created a local comkit didn't push them yet